src/MotionHandler.moon

```diff
@@ -118,7 +118,7 @@ class MotionHandler
 				.text = .text\gsub "\\pos(%b())\\t%((%d+,%d+),\\pos(%b())%)", ( start, time, finish ) ->
 					"\\move" .. start\sub( 1, -2 ) .. ',' .. finish\sub( 2, -2 ) .. ',' .. time .. ")"
 
-			@resultingCollection\addLine Line line, nil, { wasLinear: true }
+			@resultingCollection\addLine (Line line, nil, { wasLinear: true }), nil, true, true
```

Comment:
Should be @resultingCollection\addLine Line( line, nil, { wasLinear: true } ), nil, true, true

Same on L159